src/components/Medicine/MedicationAdministration/MedicineAdminForm.tsx

```
@@ -48,6 +50,150 @@ interface MedicineAdminFormProps {
+          const isSelected = idx === selectedIndex;
+          return (
+            <div
+              key={idx}
+              className={cn(
+                "flex items-start gap-3 p-3 rounded-lg border cursor-pointer transition-colors",
+                isSelected
+                  ? "bg-primary-50 border-primary-300"
+                  : "bg-gray-50 border-gray-200 hover:border-gray-300",
+              )}
+              onClick={() => handleSelectDosage(idx)}
+            >
```

Biome (2.3.14)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)